src/main/scala/org/zalando/react/nakadi/ReactiveNakadi.scala

```diff
@@ -86,6 +96,66 @@ class ReactiveNakadi {
     consumeWithOffsetSink(props, ReactiveNakadi.ConsumerDefaultDispatcher)
   }
 
+  def producerFlow(producerProperties: ProducerProperties)(implicit system: ActorSystem): Flow[ProducerMessage, Seq[String], NotUsed] = {
```

Comment:
Why implement the entire feature within the `ReactiveNakadi.scala` class. I think it may be better to encapsulate it in its own class `src/main/scala/org/zalando/react/nakadi/ProducerFlow.scala` (or something like that). And then just instantiate it here